src/sonic-pit/pit-sysdiag/src/ssd_tc.py

```diff
@@ -0,0 +1,232 @@
+import os
+import re
+import subprocess
+from test_case import TestCaseCommon
+from errcode import *
+import traceback
+
+
+def run_command(cmd):
```

Comment:
there is already a similar function(if not the same) in PR1, which shall be re-used.

Reply:
Removed in latest commit.